pkg/rke2/spw.go

```diff
@@ -42,57 +34,55 @@ func checkStaticManifests(dataDir string) cmds.StartupHook {
 	return func(ctx context.Context, wg *sync.WaitGroup, args cmds.StartupHookArgs) error {
 		go func() {
 			defer wg.Done()
+			if err := wait.ExponentialBackoff(podCheckBackoff, func() (done bool, err error) {
```

Comment:
I think at this point we have just recreated [PollImmediate](https://pkg.go.dev/k8s.io/apimachinery/pkg/util/wait#PollImmediate)
```suggestion
			if err := wait.PollImmediate(20 * time.Second, 30 * time.Minute, func() (done bool, err error) {
```